### PR TITLE
Fix for the Travis builds failing and included PHP 7.3

### PR DESCRIPTION
**Pull request by JoshuaLuckers**

### CODE REVIEW

**Comment by JoshuaLuckers** [comment]
Locally all tests seem to run fine, I have no idea why the `group by` and `having` tests are failing.

File: .travis.yml [hunk 1]
```diff
@@ -19,6 +20,9 @@ env:
 install:
   - travis_retry composer install --prefer-dist --no-interaction
 
+before_install:
+  -  if [[ "(7.0 7.1 7.2 7.3)" =~ $(phpenv version-name) ]]; then curl -sL -o ~/.phpenv/versions/$(phpenv version-name)/bin/phpunit https://phar.phpunit.de/phpunit-6.5.phar; chmod +x ~/.phpenv/versions/$(phpenv version-name)/bin/phpunit; fi
+
```
Comment (opengeek) [inline .travis.yml:25]: Why did you need to add this?

Comment (Mark-H) [reply .travis.yml:25]: Probably because a newer version of phpunit would be automatically installed, while tests are written for the older version.

Comment (opengeek) [reply .travis.yml:25]: No, it just wasn't using the path as described by Travis. It should be using the one installed by composer, but wasn't. See the alternate PR I just pushed. The problem with mysql group by and such is still there, but calling vendor/bin/phpunit directly solved the phpunit issue.

Comment (JoshuaLuckers) [reply .travis.yml:25]: I didn't consider the alternate, and better, solution. This PR is a workaround instead of the actual fix.
